_src/_posts/2018-01-04-importance-of-serving-ads.markdown

```diff
@@ -181,6 +181,9 @@ It is important for marketers to recognize the value in impressions and how they
 
 ##References
 
+[^1]: http://jampp.com/
+[^2]: We could possibly have events attributed to clicks occurring in the past.
+[^3]: https://en.wikipedia.org/wiki/Multi-objective_optimization#Scalarizing
 
 [jampp]: http://jampp.com/
 [past-click-attributed]: We could possibly have events attributed to clicks occurring in the past.
```

Comment:
You turned this named reference into a numbered one, and now it's there in both forms.

Either one has to go, or you've broken something.

Reply:
I want the references there on purpose to see how it renders. Both linked words and a compound list of references.

The uncredited change went in the second commit. I've sweeped the file over with an english dict.

Reply:
I don't get it. Uncredited seemed correct, why did you revert it?

Reply:
Because it was not part of the english dict I was using. 

But after googling it seems that `uncredited` is fine too.
https://www.collinsdictionary.com/dictionary/english/unaccredited
http://www.dictionary.com/browse/uncredited

Seems to be a matter of taste, both could go.

Reply:
The good thing is that References are now working :) 

https://geeks-preview.jampp.com/data-science/importance-of-serving-ads/

Reply:
No, I think uncredited is more accurate in this context:

https://www.google.com.ar/search?q=define:uncredited&ie=utf-8&oe=utf-8&client=firefox-b&gws_rd=cr&dcr=0&ei=ubFOWvaBCYP9wQTrvYHIDg

https://www.google.com.ar/search?q=define:unaccredited&ie=utf-8&oe=utf-8&client=firefox-b&gws_rd=cr&dcr=0&ei=trFOWoShAcKGwQTotZ-gBA

Reply:
Yes that's exactly what I did and I looked at the possible variations in english language and both seem viable given the mentioned dictionaries.

Reply:
You can freely edit if you think it does not suit proper English.